util/completion_test.go

```diff
@@ -0,0 +1,122 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+var testCommand = &cobra.Command{
+	Use: "test-command",
+}
+
+var getCompletionFnTests = []struct {
+	fName  string
+	fBody  string
+	output string
+}{
+	{
+		"test",
```

Comment:
maybe we should use the `key: "value"` form here to make things a bit more readable..

like 
```
{
fName: "test",
...
}
```